app/views/tasks/home.html.erb

```diff
@@ -1,28 +1,36 @@
-<header>
-  <h1>Welcome to your tasklist!</h1>
-</header>
+<%= render "shared/header" %>
+
 <main>
   <table>
     <tr>
       <th>Task Name</th>
       <th>Completed?</th>
       <th>Delete Task</th>
+      <th>Assigned to</th>
     </tr>
 
-  <% @all_tasks.each do |task| %>
-  <tr>
-  <td><%= link_to "#{task.name}", "show/#{task.id}" %> </td>
-    <% if task.completed_date != nil %>
-      <td> <%= "\u2713" %></td>
-    <% else %>
-      <td> <%= "\u2610" %> </td>
-    <% end %>
-  <td><%= link_to "Delete Me!", "confirm_delete/#{task.id}" %> </td>
-  </tr>
-  <% end %>
+    <tr>
+      <td>(click each for details/to edit)</td>
```

Comment:
It looks like bootstrap makes tooltips pretty easy to implement, so I will definitely keep that in mind-- I could imagine them coming in useful for bEtsy.